rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -59,7 +59,7 @@
 #define OS_IDLE_THREAD_STACK_SIZE   (MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE + EXTRA_IDLE_STACK + EXTRA_IDLE_STACK_DEBUG)
 #endif
 
-#define OS_DYNAMIC_MEM_SIZE         0
+#define OS_DYNAMIC_MEM_SIZE        1024 
```

Comment:
We know this change is wrong and are in the process of fixing the code that was dependent on this change

Reply:
Removed in 
ec81b3e